opds_import.py

```diff
@@ -596,7 +607,8 @@ def _data_detail_for_feedparser_entry(cls, entry, metadata_data_source):
             if circulation_data_source_name:
                 _db = Session.object_session(metadata_data_source)
                 circulation_data_source = DataSource.lookup(
-                    _db, circulation_data_source_name, autocreate=True
+                    _db, circulation_data_source_name, autocreate=True,
+                    offers_licenses=True
```

Comment:
If we're importing from an OPDS feed and it's a data source that's not in the list of data sources (e.g. not the metadata wrangler), it's a safe bet that it offers licenses.